app/views/layouts/_ckeditor.html.erb

```diff
@@ -0,0 +1,2 @@
+<script src="https://cdn.ckeditor.com/ckeditor5/12.3.1/classic/ckeditor.js"></script>
```

Comment:
Sử dụng gem `ckeditor` xem

Reply:
tại vì em thử rồi, mà hén bảo tạo model, với lại hén bị lỗi nhiều quá anh, nên em dùng như vậy, :)

Reply:
với lại em require ckeditor.js vô hén báo not found miết anh, nên hết cách em phải làm vậy thôi anh